src/openrct2/actions/RideDemolishAction.hpp

```diff
@@ -174,41 +174,41 @@ DEFINE_GAME_ACTION(RideDemolishAction, GAME_COMMAND_DEMOLISH_RIDE, GameActions::
             }
 
             // remove any free voucher for this ride from peep
-            if (peep->ItemStandardFlags & PEEP_ITEM_VOUCHER)
+            if (peep->GetItemFlags() & EnumToFlag(ShopItem::Voucher))
             {
                 if (peep->VoucherType == VOUCHER_TYPE_RIDE_FREE && peep->VoucherRideId == _rideIndex)
                 {
-                    peep->ItemStandardFlags &= ~(PEEP_ITEM_VOUCHER);
+                    peep->SetItemFlags(peep->GetItemFlags() & ~EnumToFlag(ShopItem::Voucher), 0);
```

Comment:
Create a `ClearItemFlags` function then you don't need to do so much repetition.